docs/tutorial/comparisons.md

```diff
@@ -211,13 +211,13 @@ result = comparison_lut[x_cast_to_uint7 - y_cast_to_uint7]
 
 #### Notes
 
-- It can result in a single table lookup as well, if x and y are assigned (because of other operations) the same bit-width, and that bit-width can store `x - y`.
-- Or in two table lookups if only one of the operands is assigned a bit-width bigger than or equal to the bit width that can store `x - y`.
+- It can result in a single table lookup, if x and y are assigned (because of other operations) the same bit-width, and that bit-width can store `x - y`.
+- Alternatively, this can be achieved in two table lookups if only one of the operands is assigned a bit-width bigger than, or equal to, the bit width that can store `x - y`.
 
 #### Pros
 
-- It will not put any constraints on bit-widths of the operands, which is amazing if they are used in other costly operations.
-- It will result in at most 3 table lookups, which is still good.
+- It will not put any constraints on the bit-widths of the operands, which is useful if they are used in other costly operations.
```

Comment:
I think we should use something stronger than useful, as we've used it for those resulting in only one constraint. I used **great** and **amazing**, we can use **useful** and **something**